src/nvim/if_cscope.c

```diff
@@ -1841,9 +1841,7 @@ static void sig_handler(int s) {
  */
 static void cs_release_csp(size_t i, int freefnpp)
 {
-  /*
-   * Trying to exit normally (not sure whether it is fit to UNIX cscope
-   */
+   // Trying to exit normally (not sure whether it is fit to Unix cscope)
```

Comment:
The linter is complaining about this line being indented 3 spaces (should be two).